### DISCUSSION TITLE
Trying to be consistent

### DISCUSSION THREAD

**Discussion by jannat226** [mrdbourke/pytorch-deep-learning#1196] [Show and tell]
Today i learnt about how the model actually works , loss function , optimizer , training loop , testing loop how to save my model and how to load it.Thank you so much for such good resources and I loved the song that you sung for the training and testing loop.
Link -> https://github.com/jannat226/pytorch-deep-learning
